transform-plugins/docs/XMLParser-transform.md

```diff
@@ -0,0 +1,82 @@
+# XML Parser Transform
+
+Description
+-----------
+The XML Parser uses XPath to extract field from a complex XML Event. This should generally be used in conjunction with
+the XML Reader Batch Source. The XML Source Reader will provide individual events to the XML Parser and the XML Parser
+will be responsible for extracting fields from the events and mapping them to output schema.
```

Comment:
The XML Parser uses XPath to extract fields from a complex XML event. This plugin should generally be used in
conjunction with the XML Reader Batch Source. The XML Reader will provide individual events to the
XML Parser, which will be responsible for extracting fields from the events and mapping them to the output schema.